Automattic/wp-calypso
# AnchorFM: Headstart Annotations

#### Changes proposed in this Pull Request

Gilbert and Riley use pub/spearhead as their underlying themes. Updating JSON and images to reflect that

#### Testing instructions

1. Apply this diff D55324-code
2. Checkout this branch or use Calypso Live
3. Go to http://calypso.localhost:3000/new?anchor_podcast={anchor_podcast_id}
4. Enter a podcast name
5. Choose a design
6. Choose a font pairing and click continue
7. Pick a site name, plan, and domain. Launch this site.
8. The website created should be flagged as as "headstart" theme with designs similar to the Gilbert or Riley theme

#### GIFs
![2021-01-13 19 16 44](https://user-images.githubusercontent.com/66652282/104540536-a5ab2900-55ed-11eb-81d2-14ae235479a4.gif)

Fixes 393-gh-Automattic/dotcom-manage
